backend/src/graphql/resolvers/BalanceResolver.ts

```diff
@@ -5,14 +5,30 @@ import { Resolver, Query, Ctx, Authorized } from 'type-graphql'
 import CONFIG from '../../config'
 import { Balance } from '../models/Balance'
 import { apiGet } from '../../apis/loginAPI'
+import { User as tUser } from '../../typeorm/entity/User'
+import { Balance as tBalance } from '../../typeorm/entity/Balance'
+import calculateDecay from '../../util/decay'
+import { roundFloorFrom4 } from '../../util/round'
 
 @Resolver()
 export class BalanceResolver {
   @Authorized()
   @Query(() => Balance)
   async balance(@Ctx() context: any): Promise<Balance> {
-    const result = await apiGet(CONFIG.COMMUNITY_API_URL + 'getBalance/' + context.sessionId)
+    // get public key for current logged in user
+    const result = await apiGet(CONFIG.LOGIN_API_URL + 'login?session_id=' + context.sessionId)
```

Comment:
Why do we login? The user is logged in already. He has a sessionID?

Reply:
The JWT Token store only the session_id so I need to call to login-server to get the public-key from the user.
The login call is for returning login-data, the user and how many transactions for the user are pending or waiting on signing. 
It is used by community-server as well and for that I have written this function long time ago.
In my previous model login via api wasn't allowed so I could use the name here. The Class for that is called JsonGetLogin.

If the JWT token would store the public key of the user, I wouldn't need calling the login-server at all in this function. 
This login function from login-server is fast because it didn't call the db. It getting the data 100% from memory. 
It would be even faster with my replaced json pr #625

Reply:
But we need the SessionID to authenticate against the Server, the publicKey is not allowing this.
Would it be possible to use `getUserInfo` instead?, since login can have unintended sideeffects + its not obvious what happens or is `getUserInfo` not providing the required data?

Reply:
For `getUserInfo` I need email but this isn't saved in JWT token either. 
The call /login which could be also translated as getDetailsForSession deliver the user object and how many transactions are pending. This call is also used from community server and therefore is good testet and hasn't any side effects. 
But I agree that the name is not quite accurate. Should I change it?

Reply:
No don't change it. I would have requested to remove the email requirement from getUserInfo anyways - we will refactor this later. Keep it as it is!